aten/src/ATen/native/quantized/cpu/kernels/QuantizedOpKernels.cpp

```diff
@@ -412,7 +412,7 @@ void qrelu_kernel(const Tensor& qx, Tensor& qy) {
         qx.q_scale(),
         qx.q_zero_point(),
         c10::nullopt);
-    using Vec = Vec256<scalar_t>;
+    using Vec = Vectorized<scalar_t>;
```

Comment:
Windows CI claims that functions like relu, relu6 are missing in the namespace Vectorized<scalar_t>. I wonder if the function implementations for some types are missing. Would you please expand scalar_t into all possible types defined in AT_DISPATCH_QINT_TYPES at a place for testing?